Gemfile

```diff
@@ -253,10 +233,4 @@ group :production, :development, :staging do
   gem 'tiny_tds'
 end
 
```

Comment:
We should test, but I _think_ removing these will cause issue with deployments, maybe only initial deployments?